nulib/meadow
# First start on Audio download button for a Fileset

# Summary 
Provide UI button for Audio Fileset download

![image](https://github.com/nulib/meadow/assets/3020266/a2df3bc3-afcc-4f89-915e-65d8b7329f8c)


# Specific Changes in this PR
- Include UI for download button for Access Audio Fileset

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
- Browse to an Audio Work which has an Access Fileset, and notice the download button.
- Button successfully downloads the audio file


# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes


# Tested/Verified
